address.go

```diff
@@ -110,8 +110,8 @@ const (
 
 // New creates a new address using the AddressMessage information.
 func (a *AddressService) New(req *AddressMessage) error {
-	flags := netlink.Request
-	_, err := a.c.Send(req, RTM_NEWADDR, flags)
+	flags := netlink.Request | netlink.Create | netlink.Acknowledge | netlink.Excl
+	_, err := a.c.Execute(req, RTM_NEWADDR, flags)
```

Comment:
Are these changes needed for this fix?
I am planning to do a rewrite to make use of `netlink.Execute` instead of `netlink.Send` for concurrency reasons, but am waiting to implement is library wide. If not needed I would prefer not to merge this (yet).

Reply:
I was running in to an issue with `Send` where another call (from within the process) would be receiving the response; however, it was last week some time, so I know long remember the details. `Send` alone might work if we drop the `Acknowledge` flag?